core/src/cg/group4/view/screen/CollectiblesScreen.java

```diff
@@ -229,33 +225,25 @@ public void display() {
      */
     protected void constructContents() {
         Gdx.app.log(getClass().getSimpleName(), "Construction");
-
-        ArrayList<Collectible> sortedList = cSelectedCollection.sort(cSorter);
-        DecimalFormat format = new DecimalFormat("#.00");
+        final Player player = StandUp.getInstance().getPlayer();
+        DecimalFormat format = new DecimalFormat("0.##");
 
         boolean myCollection = cGroupsBox.getSelectedIndex() == 0
```

Comment:
I though about it, but because i only use the boolean in a single spot. I thought using three or four variables would be overkill.